jenkins/ci.opensuse.org/openstack-prepare-staging.yaml

```diff
@@ -15,3 +15,7 @@
       - build-name:
           name: '#${BUILD_NUMBER} / C:O:${ENV,var="openstack_project"}'
       - timestamps
+      - timeout:
```

Comment:
You should use the `type: no-activity` timeout strategy instead of the default `absolute` strategy, to have the timeout value applied to the amount of time that the job spends with no output in the build log. This allows you better control over your timeout value, because you don't need to account for the amount of time it usually takes the full job to complete.

Reply:
It should time out in this case even when output happens.

Reply:
Ok, as long as that is your intention.